src/fsharp/vs/ServiceUntypedParse.fs

```diff
@@ -340,8 +340,13 @@ type FSharpParseFileResults(errors : FSharpErrorInfo[], input : Ast.ParsedInput
  
         ErrorScope.Protect 
             Range.range0 
-            (fun () -> findBreakPoints() |> List.tryLast)
-            (fun _msg -> None)   
+            (fun () -> 
+                let locations = findBreakPoints()
+                
+                match locations |> List.filter (fun m -> rangeContainsPos m pos) with
+                | [] -> Seq.tryHead locations
```

Comment:
No that's for when you click on the marker on the left border. Then post is zero and therefore it will never be inside an expression. But as I said I plan to clean this up today.